ansible/roles/networking/tasks/main.yaml

```diff
@@ -57,6 +57,39 @@
     - disable_iptables | default(true)
     - iptables_exists.rc == 0
 
+- name: ensure NetworkManager's conf.d directory exists
+  file:
+    path: /etc/NetworkManager/conf.d
+    state: directory
+
+# prevent NetworkManager to create conflicting entries in the routing table.
+# https://projectcalico.docs.tigera.io/maintenance/troubleshoot/troubleshooting#configure-networkmanager
+- name: configure network manager to prevent it from interfering with calico interfaces
```

Comment:
moved this task out to apply it to all OS family as it is not OS specific according to the docs.